src/teleop/jetson/arm_trajectory_server.py

```diff
@@ -122,8 +164,25 @@ def execute_cb(self, goal: FollowJointTrajectoryGoal) -> None:
                 self._feedback.actual.positions = joint_states.position
                 self._feedback.actual.velocities = joint_states.velocity
                 self._feedback.actual.time_from_start = rospy.Time.from_sec(time.time()) - time_start
+            self._feedback.error.positions = [
+                desired - actual
+                for desired, actual in zip(self._feedback.desired.positions, self._feedback.actual.positions)
+            ]
+            self._feedback.error.velocities = [
+                desired - actual
+                for desired, actual in zip(self._feedback.desired.velocities, self._feedback.actual.velocities)
+            ]
             self._as.publish_feedback(self._feedback)
             # ---------------------------------------
+            # Abort upon exceeding error threshold
+            error = error_threshold_exceeded(self._feedback)
+            if error:
+                self._result.error_code = -4  # PATH_TOLERANCE_VIOLATED
```

Comment:
If you google for FollowJointTrajectoryResult you'll find it.
https://docs.ros.org/en/diamondback/api/control_msgs/html/msg/FollowJointTrajectoryResult.html

But I also don't remember how the next line of code works since according to that link, there's no error_string attribute.

Reply:
Here's the correct (I think) documentation.
https://docs.ros.org/en/noetic/api/control_msgs/html/action/FollowJointTrajectory.html